index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="main-content">
+      <nav>
+        <span>Muhyideen Elias</span>
+        <img src="./images/menu.svg" alt="menu" />
+      </nav>
```

Comment:
- [ ] Please enclose this in a `header` tag. The navbar will always be at the top of the page and it is always semantic to enclose it in a a `header` tag 👍

Reply:
@Lucas-Masaba I think it is browser incompatibility, it was okay on Safari but forgot to test on other browsers. okay make necessary changes, test, and revert. thanks, soo much.

Reply:
- Hello @Fourteen98  I thought so. I just wasn't sure since I do not have any other browser open. Try to test it on other browsers and see if it works fine